include/osg/StateAttribute

```diff
@@ -122,6 +122,7 @@ class OSG_EXPORT StateAttribute : public Object
           * that type's value. */
         enum Type
         {
+            IMAGE,
```

Comment:
New StateAttribute types should be appended to the enum list, as the values are important for binary compatibility and the serialises.